utils/rest_client.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
As far as I can tell, these are part of the management system. Using them directly (and not through a mgmt_sys) is not something we should do since we'll always go through the mgmt_sys to get them. The exceptions should probably be in `utils.mgmt_system.exceptions` and the rest should live in the same place as the container provider base class.